tests/models/test_image.py

```diff
@@ -1365,7 +1365,7 @@ def test_image_products_are_deleted(ptf_datastore, data_dir, archive):
     # make sure the files are there
     local_files = []
     archive_files = []
-    for obj in [im, im.psf, im.sources]:  # TODO: add WCS when it becomes a FileOnDiskMixin
+    for obj in [im, im.psf, im.sources, im.wcs]:  # TODO: add WCS when it becomes a FileOnDiskMixin
```

Comment:
todo: remove comment if looks good

Reply:
Yes remove this comment. 
In general don't worry too much about removing things, they all show up in the diff so you can always bring them back if you need to.